README.md

```diff
@@ -74,7 +74,7 @@ const config = {
 - Default: `webpackConfig.output.publicPath`
 - Reference: [webpack publicPath](https://webpack.docschina.org/configuration/output/#outputpublicpath)
 
-使用独立的 publicPath，一般情况下不需要设置此参数。
+Using publicPath, you do not normally need to set this parameter.
```

Comment:
`To use independent publicPath, you need not to set this parameter under normal circumstances.`